tests/lib/c_lib/thrd/prj.conf

```diff
@@ -7,3 +7,4 @@ CONFIG_THREAD_STACK_INFO=y
 CONFIG_DYNAMIC_THREAD=y
 CONFIG_DYNAMIC_THREAD_POOL_SIZE=2
 CONFIG_COMMON_LIBC_THRD=y
+CONFIG_MP_MAX_NUM_CPUS=1
```

Comment:
If it's failing on any platform sporadically, that should be a bug. Can you file an issue as well and mention that information?

A comment above this line referencing the GitHub issue would be preferable as well, since this option should be removed when the bug is fixed.

Reply:
good that you mention it, see 
https://github.com/zephyrproject-rtos/zephyr/issues?q=sort%3Aupdated-desc%20is%3Aissue%20state%3Aclosed%20tests%2Flib%2Fc_lib%2Fthrd

this test has been failing for months, I thought it was a CI thing, but was able to reproduce this easily. Anyways, I will open a new one and make it high priority and hope it will get fixed soon.